src/terminal/frame.rs

```diff
@@ -97,13 +118,42 @@ impl Frame<'_> {
     /// ```
     ///
     /// [`Layout`]: crate::layout::Layout
+    #[stability::unstable(feature = "widget-ref")]
```

Comment:
This should go on the `render_stateful_widget_ref` method instead, right?

Reply:
yep